github4s/src/main/scala/github4s/interpreters/IssuesInterpreter.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2016-2020 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package github4s.interpreters
+
+import github4s.http.HttpClient
+import github4s.algebras.Issues
+import github4s.GithubResponses.GHResponse
+import github4s.domain._
+import java.net.URLEncoder.encode
+import github4s.Decoders._
+import github4s.Encoders._
+
+class IssuesInterpreter[F[_]](implicit client: HttpClient[F], accessToken: Option[String])
+    extends Issues[F] {
```

Comment:
Based on my comment above, we could remove this `extends`. If that makes sense, we could also provide a companion object. Something like this:

```scala
class Issues[F[_]](implicit client: HttpClient[F], accessToken: Option[String]) { ... }

object Issues {
  def apply[F[_]](implicit client: HttpClient[F], accessToken: Option[String]): Issues[F] = new Issues[F]
}
```